layouts/shortcodes/attestation-letter.md

```diff
@@ -0,0 +1,3 @@
+The Attestation Letter is a one-page report that you can share with external stakeholders 
+such as prospects or customers. We base the letter on our [Executive Summary](/getting-started/what-to-expect/report-contents/#executive-summary).
+You cannot customize an Attestation Letter.
```

Comment:
Not sure if it's worth mentioning this across all the formats. But if you've got variables for the other formats, it couldn't hurt to make the "you can't customize this" note there too.

Reply:
I don't (yet) have variables for the other formats :)